Makefile

```diff
@@ -18,6 +18,28 @@ COCKPIT_REPO_STAMP = pkg/lib/cockpit.js
 
 all: $(WEBPACK_TEST)
 
+# checkout common files from Cockpit repository required to build this project;
+# this has no API stability guarantee, so check out a stable tag when you start
+# a new project, use the latest release, and update it from time to time
```

Comment:
I moved this code to the top of the file, as variables must be declared before use.